Formula/wasmtime.rb

```diff
@@ -26,6 +27,11 @@ class Wasmtime < Formula
 
   def install
     system "cargo", "install", *std_cargo_args
+    system "cargo", "rustc", "--release", "-p", "wasmtime-c-api"
```

Comment:
can't `cargo install` do this?

Reply:
In some other formulae, we use `cargo build` instead.

https://github.com/Homebrew/homebrew-core/blob/6af596994fe668c1e21c2f78bd65b3b03381eecb/Formula/resvg.rb#L25

I'm not entirely familiar with the differences, but I think some consistency would be desirable, assuming that there's no other reason to deviate.

Reply:
I don't see an obvious way for `cargo install` to build it.

I chose to use `cargo rustc` since it seemed to be more a bit clearer to specify which package to build, instead of providing a manifest path manually.

I agree that being consistent might be a better option.  Updated the patch to reflect it.

Reply:
Ping?